projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -59,18 +87,13 @@ frontend-shared-components-release:
   before_script:
     - cd projects/frontend/shared-components/gui
   script:
-    - ../../cicd/build_shared.sh
```

Comment:
Where does the package we publish to npm registry come from if we don't run the build script?

Reply:
Ah, nvm, just saw that you run the build stage on merge to main as well.

Reply:
The package is built once during build stage, then the artifact is reused by other jobs (until expired in 7 days).